sirepo/job_driver/sbatch.py

```diff
@@ -153,19 +153,30 @@ async def _do_agent_start(self, op):
         def write_to_log(stdout, stderr, filename):
             p = pkio.py_path(self._local_user_dir).join('agent-sbatch', self.cfg.host)
             pkio.mkdir_parent(p)
-            pkjson.dump_pretty(
-                PKDict(stdout=stdout, stderr=stderr),
-                p.join(f'{datetime.datetime.now().strftime("%Y%m%d%H%M%S")}-{filename}.log'),
-            )
-
-        async def get_agent_log(connection):
-            await tornado.gen.sleep(self.cfg.agent_log_read_sleep)
-            async with connection.create_process(
-                f'/bin/cat {agent_start_dir}/{log_file}'
-            ) as p:
-                o, e = await p.communicate()
-                write_to_log(o, e, 'remote')
+            f = p.join(f'{datetime.datetime.now().strftime("%Y%m%d%H%M%S")}-{filename}.log')
+            d = PKDict(stdout=stdout, stderr=stderr)
+            pkjson.dump_pretty(d, f)
+            if pkconfig.channel_in('dev'):
+                pkdlog(d.pkupdate(filename=f))
```

Comment:
do we not want to log the file name always?

Reply:
I think it is redundant. In cases where we don't log it then we'll be reading from the file so we'll know the name. But, it doesn't hurt 9d59f2800